<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWin</class>
 <widget class="QMainWindow" name="MyMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMainWin</string>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">
border-image: url(:/images/UI5.jpg);


QWidget#rollWidget{
background-color: rgb(85, 255, 127);
}

QWidget#weatherWidget{
background-color: rgb(255, 170, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="WeathWin" name="weatherWidget" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="PictureWin" name="pictureWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 170, 255);</string>
    </property>
   </widget>
   <widget class="VideoWin" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 170, 255);</string>
    </property>
   </widget>
   <widget class="RollWin" name="rollWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>800</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RollWin</class>
   <extends>QWidget</extends>
   <header>rollwin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoWin</class>
   <extends>QWidget</extends>
   <header>videowin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeathWin</class>
   <extends>QWidget</extends>
   <header>weathwin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PictureWin</class>
   <extends>QWidget</extends>
   <header>picturewin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
